test/sequential/test-watch-mode.mjs

```diff
@@ -791,4 +791,54 @@ process.on('message', (message) => {
       `Completed running ${inspect(file)}`,
     ]);
   });
+
+  it('when multiple `--watch` flags are provided should run as if only one was', async () => {
```

Comment:
Note: maybe node should error if multiple `--watch` flags are provided?

I did not go with such a solution because that would be a breaking change, just ignoring extra `--watch` flags is a non breaking change and it solves the issue at hand

If someone believes that multiple `--watch` flags should cause an error I would propose to first land this change right now as a patch then as a followup I can also add the stricter validation which will be included as a breaking change in the next major 🙂

Reply:
And/or maybe, the erroring when multiple `--watch` flags are provided is part of the bigger issues of node being quite permissive with its cli flags: https://github.com/nodejs/node/issues/57864 and could be addressed as part of that 🤔